selfdrive/car/toyota/carcontroller.py

```diff
@@ -239,7 +242,7 @@ def update(self, sendcan, enabled, CS, frame, actuators,
       send_ui = False
 
     if (frame % 100 == 0 or send_ui) and ECU.CAM in self.fake_ecus:
-      can_sends.append(create_ui_command(self.packer, steer, sound1, sound2))
+      can_sends.append(create_ui_command(self.packer, steer, sound1, sound2, 0, 2 - left_line, 2 - right_line))
```

Comment:
why sending `2 - xxx_line` and not simply 'xxx_line'?

Reply:
0 is for lane detection completely off. 1 and 2 are for solid line and line not seen. But from the logic it is reversed. Having it only send xxx_line shows off when it skills show on.

Reply:
ah ok. So makes sense to send 2 when line is not detected.